src/utils/config.ts

```diff
@@ -69,6 +68,21 @@ const JWTParams = t.union([
 ]);
 type JWTParams = t.TypeOf<typeof JWTParams>;
 
+const AttributeAuthorityParams = t.union([
+  t.interface({
+    ENABLE_ADE_AA: t.literal(true),
+
+    ADE_AA_API_ENDPOINT: NonEmptyString,
+    ENDPOINT_L1_SUCCESS: NonEmptyString,
+    L1_TOKEN_EXPIRATION: NonNegativeInteger,
+    L2_TOKEN_EXPIRATION: NonNegativeInteger
+  }),
+  t.interface({
+    ENABLE_ADE_AA: t.literal(false)
+  })
```

Comment:
Can we add `TOKEN_EXPIRATION` (now `DEFAULT_TOKEN_EXPIRATION`) in here?
`L1_TOKEN_EXPIRATION ` and `L2_TOKEN_EXPIRATION ` are required if `ENABLE_ADE_AA ` is `true`
`TOKEN_EXPIRATION` is required when `ENABLE_ADE_AA` is false.
Right?
```suggestion
  t.interface({
    ENABLE_ADE_AA: t.literal(false),
    TOKEN_EXPIRATION: NonNegativeInteger
  })
```